### PR TITLE
Added extra safety checks in "isComplete" for #474

### PR DESCRIPTION
**Pull request by SabbeRubbish**
As per your request :-)
Linked to issue #474

### CODE REVIEW

File: js/jquery.inputmask.js [hunk 1]
```diff
@@ -1645,10 +1645,13 @@
...
-                        activeMasksetIndex = this.data('_inputmask')['activeMasksetIndex'];
-                        opts = this.data('_inputmask')['opts'];
-                        return maskScope(masksets, activeMasksetIndex, opts, { "action": "isComplete", "buffer": this[0]._valueGet().split('') });
+						if (this.data('_inputmask')) {
+							masksets = this.data('_inputmask')['masksets'];
+							activeMasksetIndex = this.data('_inputmask')['activeMasksetIndex'];
+							opts = this.data('_inputmask')['opts'];
+							return maskScope(masksets, activeMasksetIndex, opts, { "action": "isComplete", "buffer": this[0]._valueGet().split('') });
+						}
+						else return true;
```
Comment (SabbeRubbish) [inline js/jquery.inputmask.js:1654]: Not sure if it should return true or false.  I'm guessing true as missing masks or no configuration makes no sense to mark it as "incomplete".

**Comment by RobinHerbots** [comment]
@SabbeRubbish , 

To be 100% correct we should return undefined, but for validation return true will be more desired I think.

Thx
Robin

**Comment by thiagotalma** [comment]
Always remember to upgrade the dist files ;-)

**Comment by SabbeRubbish** [comment]
As a noob Gitter is supposed to do, he forgets :-)
Btw, I'd have to work myself in the bundling thing, which I have not time for at the moment.

**Comment by RobinHerbots** [comment]
No probs, what did work on for the bundling thing?

**Comment by SabbeRubbish** [comment]
Just haven't had the time for it. Plus, wouldn't know what bundling tool you use for the minification e.g.

**Pull request merged**